hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson

```diff
@@ -2,59 +2,8 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
+  import_cfgs: ["{proj_root}/hw/dv/tools/dvsim/testplan/tl_device_access_types_wo_intg_testplan.hjson"]
```

Comment:
This needs to be `import_testplans`. 
https://github.com/lowRISC/opentitan/blob/2f2a5fd4e32aca76082537b7396e35b2039f4bdd/util/dvsim/Testplan.py#L424

Reply:
oh, good catch. Fixed, thanks